CONTRIBUTING.md

```diff
@@ -70,3 +71,32 @@ This project uses [cSpell](https://cspell.org/) for spell checking. The spellche
 Words may be globally allowed or globally blocked by adding to the [cspell/](./cspell/) folder. See the [cSpell documentation on dictionaries](https://cspell.org/docs/dictionaries/) for more info.
 
 To override instances without adding to the list, either add a `// cSpell: disable` comment or `// cSpell:words [list of comma-separated words]` comment. See the [cSpell documentation on overrides]https://cspell.org/docs/Configuration/document-settings#words) for more info.
+
+## Publishing a report
```

Comment:
Adds a rough guide to CONTRIBUTING.md to describe the process of cutting a new release. It’s still a little manual, intentionally.